refinery/tool_manager/models.py

```diff
@@ -31,22 +33,28 @@ class Parameter(models.Model):
     uuid = UUIDField(unique=True, auto=True)
     name = models.TextField(max_length=100)
     description = models.TextField(max_length=500)
-    is_user_adjustable = models.BooleanField(default=False)
+    is_user_adjustable = models.BooleanField(default=True)
```

Comment:
I thought it made sense now that we have moved away from generating Parameter objects for every parameter included in each of a Workflow's steps to only exposing a .step's parameters that have been properly annotated. If someone takes the time to properly annotate, I think by default they would like to be able to adjust their parameters in the UI